module/zcommon/zfs_prop.c

```diff
@@ -359,6 +359,8 @@ zfs_prop_init(void)
 	    PROP_DEFAULT, ZFS_TYPE_VOLUME, "<zvol worker>", "NWORKER");
 	zprop_register_string(ZFS_PROP_REPLICA_ID, "io.openebs:zvol_replica_id",
 	    "", PROP_DEFAULT, ZFS_TYPE_VOLUME, "<replica id>", "REPLICA_ID");
+	zprop_register_string(ZFS_PROP_ZVOL_READONLY, "io.openebs:readonly", "",
```

Comment:
@vishnuitta @mynktl , io.openebs:readonly does not convey it's meaning, also it conflicts with the existing readony property. Can we rename it to io.openebs:suspend (or some better name)?

Reply:
both property have the same functionality but at different layer. To differentiate this i thought of giving a different name. One more point, this is completely limited to cstor API layer only, so better to group it with organization.